src/init_python.cpp

```diff
@@ -38,14 +38,12 @@ int init_python( int argc, char *argv[] )
 
 // Q: What can argc, argv be used for?
 // A: Probably can encode some settings, but since we aren't use them, comment them out.
-   // wchar_t **wchar_t_argv = (wchar_t **) malloc(argc * sizeof(wchar_t *));
-   // wchar_t wchar_temp[1000];
-   // for (int i = 0; i < argc; i = i+1) {
-	  // printf("argv[%d] = %s\n", i, argv[i]);
-   //    mbtowc(wchar_temp, argv[i], 1000);
-   //    wchar_t_argv[i] = wchar_temp;
-   // }
-   // PySys_SetArgv( argc, wchar_t_argv );
+   wchar_t **wchar_t_argv = (wchar_t **) malloc(argc * sizeof(wchar_t *));
+   for (int i = 0; i < argc; i = i+1) {
+	  printf("argv[%d] = %s\n", i, argv[i]);
+      wchar_t_argv[i] = Py_DecodeLocale(argv[0], NULL);;
```

Comment:
@matthewturk 
Hi, I think we need to take good care of this, because it suggests that we shouldn't called this API directly (since [Py3.10](https://docs.python.org/3.10/c-api/sys.html#c.Py_DecodeLocale)) and [PyConfig](https://docs.python.org/3.8/c-api/init_config.html#pyconfig) is introduced since Py3.8.
Unless there is an urgent case to use this, I think it is better to leave it commented or to consider different Python versions.

Reply:
OK, I'm fine with that!  Please feel free to close the PR.